src/fidesops/service/connectors/query_config.py

```diff
@@ -392,6 +392,19 @@ def get_formatted_update_stmt(
         return f'UPDATE "{self.node.address.collection}" SET {",".join(update_clauses)} WHERE  {" AND ".join(pk_clauses)}'
 
 
+class RedshiftQueryConfig(SQLQueryConfig):
+    """Generates SQL in Redshift's custom dialect."""
+
+    def get_formatted_query_string(
+        self,
+        field_list: str,
+        clauses: List[str],
+    ) -> str:
+        """Returns a query string with double quotation mark formatting for tables that have the same names as
+        Redshift reserved words."""
```

Comment:
> same names as Redshift reserved words

This is a great edge case, thanks. Is there a way to programmatically know which words are reserved without looking at the docs? Say, if it was stored in a meta schema

Reply:
I haven't figured out how to do that yet, but there is `from sqlalchemy_redshift.dialect import RESERVED_WORDS`

Reply:
This is awesome. They have this for Snowflake too https://github.com/snowflakedb/snowflake-sqlalchemy/blob/master/base.py#L19 which I'll make a note to add retroactively